README.md

```diff
@@ -12,7 +12,39 @@ Please tag **@edx/educator-neem** on any PRs or issues.
 
 The front-end of our editable Gradebook feature.
 
-## Usage
+### What does this offer over the stock LMS Gradebook?
+
+![A screenshot of the grade listings](documentation/screenshots/grade-listings.png)
+
+The MFE offers a great deal more granularity when searching for problems, an easy interface for editing grades, an 
```

Comment:
It would be better to spell out "micro-frontend" where you can.

Reply:
Changed.